modyn/evaluator/internal/metrics/roc_auc.py

```diff
@@ -26,6 +26,11 @@ def _dataset_evaluated_callback(self, y_true: torch.Tensor, y_pred: torch.Tensor
         """
         assert self.evaluation_result is None
 
+        if y_pred.dim() < 1 or y_true.dim() < 1 or len(y_pred) < 2 or len(y_true) < 2:
```

Comment:
y_pred / true shape should match (see next assert) so its enough to check one of them